File: fizzbuzz.js
var FizzBuzz = function(n){
  var answer = "";
  if(!(n%3)) answer = answer + "Fizz";
  if(!(n%5)) answer = answer + "Buzz";
  if(!answer.length) answer = n;
  return answer;
};

for(var i = 1; i <= 100; i++){
  console.log(FizzBuzz(i));
}

File: fizzbuzz.rb
def fizzbuzz(n)
  answer = ""
  answer << "Fizz" if (n%3).zero?
  answer << "Buzz" if (n%5).zero?
  answer = n       if answer.length.zero?
  answer
end

(1..100).each do |i|
  p fizzbuzz i
end
